### DISCUSSION TITLE
Field conditions not working in JCB 2.12.9?

### DISCUSSION THREAD

**Discussion by dformdotdk** [joomengine/Joomla-Component-Builder#705] [Q&A]
Hi

In the latest JCB 2.12.9 I'm having issues using field conditions in my admin view. See attached screenshots. 
I'm trying to show my field "discount_code" only if "discount_type" is 0 (Manuel). discount_type is a list type. 

Anyone else having this issue?

<img width="1913" alt="Skærmbillede 2021-03-22 kl  16 30 51" src="https://user-images.githubusercontent.com/6020984/112015624-46592000-8b2c-11eb-8ba4-2d7422db85ba.png">
<img width="809" alt="Skærmbillede 2021-03-22 kl  16 30 26" src="https://user-images.githubusercontent.com/6020984/112015632-48bb7a00-8b2c-11eb-99da-827512347b73.png">

**Comment by Llewellynvdm** [comment]
Hey, please read my comments on this https://github.com/vdm-io/Joomla-Component-Builder/issues/713#issuecomment-808915018 issue. As it explains how I would recommend using the field conditions.

Then in this case, `show only` may not mean to you what I intended when I created it. Try variations... I suspect your looking for `show toggle` as the `show only` feature will only show the field if another condition has hidden it, it does not hide the field, it literally "Shows only". Now English is my second language, and so... tell me what will make more sense :+1: and we change it.

The problem we had, is we needed a way to do very complex combinations of conditions where some conditions tries to hide a field and other tries to show it, and the original why was just a toggle option that either toggle from hidden to show or from show to hidden. These would fight, and we had a hard time to get some of these conditions to work. So I realized we needed the option to let a condition only trigger one way, and not toggle... between show and hide. But just hide or just show... and that is what those conditions do... so on that condition A `show only` if the field is hidden by another condition B, this condition A will cause it to show provided that, that condition B is `hide only`. This means they play well together unlike the toggle options. The toggle options work much like Joomla's own `showon` feature.

Anyway... my best advice it to always play around a little with the various options, and look at the code it writes. Do remember I am a back-end developer who writes JS in PHP scripting style... lol
